### PR TITLE
Add Carlos Santana 🎸 as SC election 2020 candidate

### PR DESCRIPTION
**Pull request by csantanapr**
Per SC election process, here is a PR with my candidacy form.

[Rendered](https://github.com/csantanapr/knative-community/blob/ksc-nomination-cs/elections/2020/SC/csantanapr.md)

### CODE REVIEW

**Comment by jberkus** [comment]
This candidate submission meets all requirements.

@bsnchan, please merge

File: elections/2020/SC/csantanapr.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* Initial contributions in design and architecture to eventing
+* Speaker in conferences, meetups and online events for Knative (ie ibm-think, kubecon, oss, apachecon, knative, ibm-developer, etc)
+* Maintain many tutorials used for presentations [knative-minikube](https://github.com/csantanapr/knative-minikube), [knative-kind](https://github.com/csantanapr/knative-kind), [knative-tekton](https://github.com/csantanapr/knative-tekton), [knative-pi](https://github.com/csantanapr/knative-pi)
+* Contributions to docs and samples
+* Active awareness of knative resources and activities in twitter and slack.
+* Co-started weekly Friday Office/Hacky Hours
+
+## Why You're Running for SC (max 300 words)
+
+I want to contribute my time, energy, and passion on growing a healthy community for Kantive.
```
Comment (vaikas) [inline elections/2020/SC/csantanapr.md:19]: nit: Kantive -> Knative :) Marks fave typo :)

Comment (csantanapr) [reply elections/2020/SC/csantanapr.md:19]: Thanks for catching the typo, sent PR #369

**Pull request merged**